### DISCUSSION TITLE
Supabase flutter 1.0 release schedule.

### DISCUSSION THREAD

**Discussion by hhamja** [supabase/supabase#8777] [Questions] [answered]
When will supabase flutter 1.0 be officially released? 
I am very interested. 

 please let me know

**Comment by dshukertjr** [comment, marked as answer]
Thanks for your strong interest in supabase-flutter v1.0! Since supabase-js v2.0 was released yesterday, supabase-flutter will be released soon, hopefully this week!

**Reply by dshukertjr** [reply]
And for anyone landing here. supabase-flutter v1.0 is released.
https://supabase.com/blog/supabase-flutter-sdk-v1-released
